planet/clients/orders.py

```diff
@@ -80,14 +74,12 @@ def __init__(
 
     @staticmethod
     def _check_order_id(oid):
-        msg = f'Order id ({oid}) must be a valid UUID string.'
-        if not isinstance(oid, str):
-            raise OrdersClientException(msg)
-
+        """Raises planet.exceptions.ValueError if oid is not a valid UUID"""
```

Comment:
Reusing the name of a built-in exception could lead to problems in users' code.

Reply:
How about just an OrderError for almost everything order related? Until we learn that more specific exceptions are really needed.

Reply:
Yeah, it could lead to problems. I agree. I would actually throw a OEM `ValueError`, but I'm trying to have all exceptions derive from `PlanetError`. My preference here is to have an exception that handles "invalid argument type" cases and is derived from `PlanetError` vs an exception that is specific to the Orders client. Just because this type of thing is likely to show up in all of the clients. What about `ValidationError(PlanetError)`?

Reply:
Ohh! Found an example in the wild - [botocore.exceptions.ValidationError](https://github.com/boto/botocore/blob/21b06efb39da61fe4101ab9709140dc1e51dc33d/botocore/exceptions.py#L294)